CMakeLists.txt

```diff
@@ -12,6 +12,10 @@ configure_file(tinyxml2_vendorConfig-version.cmake.in
 
 install(DIRECTORY cmake DESTINATION share/${PROJECT_NAME})
 
+install(FILES
+  "${CMAKE_CURRENT_SOURCE_DIR}/package.xml"
```

Comment:
The source files to be installed are by default relative to `CMAKE_CURRENT_SOURCE_DIR` so you can remove the use of that variable and just mention `package.xml`.

Reply:
Done, thanks.